README.rst

```diff
@@ -30,7 +30,7 @@ Demo (Click to View)
 |asciicast|
 
 .. |asciicast| image:: https://asciinema.org/a/42968.png
-   :target: https://asciinema.org/a/42968?autoplay=1
+   :target: https://asciinema.org/a/42968
```

Comment:
the issues problem was not with the autoplay right? **she clicked to view** it was the fact that it was too fast and without narration. autoplay is fine IMO it attracts attention.